collect/cnrfc/cnrfc.py

```diff
@@ -827,8 +872,10 @@ def esp_trace_analysis_wrapper():
 
 def _apply_conversions(df, duration, acre_feet, pdt_convert, as_pdt):
 
-    # convert kcfs/day to cfs/day
-    df = df * 1000.0
+    # convert kcfs/day to cfs/day (unless stage data)
+    filter_columns = [x for x in df.columns if x[1] != 'SSTG']
```

Comment:
Let's add a helper function that records the units for the different forecast codes.  CNRFC added these helpful notes to https://www.cnrfc.noaa.gov/deterministicHourlyProductCSV.php
>Note 1: Data columns labeled QINE and SQIN in the files represent flows in kcfs (thousands of cubic feet per second).
>Note 2: Data columns labeled SSTG in the files represent stages in feet.